Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -976,6 +983,48 @@ def get_incident_command(client: Client, args: dict[str, Any]) -> CommandResults
     return command_results
 
 
+def list_external_websites_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    list_external_websites command: Get external websites .
+
+    Args:
+        client (Client): CortexXpanse client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``.
+            ``args['filter']`` Used for filter websites based on authentication type
+            ``args['limit']`` Used for limit num of results
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``
+    """
+    limit = int(args.get('limit', DEFAULT_SEARCH_LIMIT))
+    searchFilter = args.get('filter')
```

Comment:
```suggestion
    searchFilter = args.get('authentication')
```
forgot to change this